### DISCUSSION TITLE
concrete_uniax

### DISCUSSION THREAD

**Discussion by dethoff** [woodem/woo#43] [Q&A]
Hi,
I am new to woo and was trying to use the example script concrete_uniax.py for getting started. However, after the definition:
```
S.lab.topImpose,S.lab.botImpose=VelocityAndReadForce(dir=Vector3.UnitZ,invF=False),VelocityAndReadForce(dir=Vector3.UnitZ,invF=False)
```

I get the following error, once I want to access `S.lab.topImpose.sumF`:
```
Traceback (most recent call last):
   File "/home/fabian/.local/lib/python3.9/site-packages/wooMain.py",
line 377, in runScript
     exec(open(script,'r').read(),globals())
   File "<string>", line 112, in <module>
   File "<string>", line 111, in start
TypeError: Unable to convert function return value to a Python type!
The signature was  (arg0: woo.dem.VelocityAndReadForce) -> OpenMPAccumulator<double>
```
Is there a convenient way to convert the OpenMPAccumulator into a python type?

Kind regards,
Fabian

**Comment by eudoxos** [comment]
Hi Fabian, sorry for the delayed reply. This used to work previously and now does not as a result of migration from `boost::python` to `pybind11`. Let me see what can be done.

**Reply by eudoxos** [reply]
Please have a look at the latest revision f170ca4b9443a23b8ba6e28504ce84dbad806e75 , `OpenMPArrayAccumulator<...>` should now appear as list when read from Python.
